backend/api/Services/EchoService.cs

```diff
@@ -131,13 +152,23 @@ private List<EchoTag> ProcessPlanItems(List<PlanItem> planItems)
 
             foreach (var planItem in planItems)
             {
+                if (planItem.PoseId is null)
+                {
+                    string message = $"Invalid EchoMission: {planItem.Tag} has no associated pose id.";
+                    _logger.LogError(message);
```

Comment:
I see that most other services don't have loggers and the logging is handled in the controller. Should we do the same for the EchoController?

Reply:
Not sure, maybe the other way around? What do you think @Christdej?

Reply:
I think generally it makes sense to log where the exception is handled and exceptions should be handled in as high a level as possible

Reply:
As long as we are consistent Im happy, so move it to the controller yes